spectests/structs.go

```diff
@@ -111,7 +111,7 @@ type SigningRoot struct {
 }
 
```

Comment:
Did you add the ``merge`` tests?

Reply:
No I didn't. Any recommendation for how to go about doing that? The multi-fork spectest support over in #60 hasn't gone anywhere, I could prepare a PR to separate that from the other changes in #60 and then we can use that as a base for adding additional spectests. Or let me know if you prefer a different approach.